specification/hardwaresecuritymodules/HardwareSecurityModules.Management/back-compatible.tsp

```diff
@@ -4,7 +4,7 @@ using Azure.ClientGenerator.Core;
 using Microsoft.HardwareSecurityModules;
 
 #suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
-@@flattenProperty(BackupResult.properties);
+@@flattenProperty(BackupResult.properties, "!python");
```

Comment:
@ChenxiJiang333 this package is not GAed yet, so python don't want to introduce flatten. Please help update for other flatten part.